src/HtmlReader/effects.ts

```diff
@@ -94,7 +94,79 @@ export function setCss(html: HTMLElement, settings: ReaderSettings): void {
     familyToReadiumFamily[settings.fontFamily]
   );
   setCSSProperty(html, '--USER__fontSize', `${settings.fontSize}%`);
+
+  // GLOBAL
   setCSSProperty(html, 'overflow', settings.isScrolling ? 'scroll' : 'hidden');
   // set the number of columns to only ever have 1.
   setCSSProperty(html, '--USER__colCount', '1');
 }
+
+/**
+ * Apply the transform property to the iframe document to fit the current screen viewport.
+ */
+export function setFXLCss(
```

Comment:
It might make more sense to call this function in the setCss function actually and detect the book layout and apply the CSS to the page, what do you think? @kristojorg 

Then the setCss function will accept a few more parameters (layout, viewport element)